ts/DataGrid/Header/HeaderCell.ts

```diff
@@ -128,7 +128,9 @@ class HeaderCell extends Cell {
         }, this.htmlElement);
         this.contentWrapper = makeHTMLElement('span', {}, this.headerContent);
 
-        if (userOptions.useHTML) {
+        const isHTML = this.value?.toString().indexOf('<') === -1;
```

Comment:
Should be !==

Perhaps make a util function of this, since we are using it multiple places? Will be more in the future.